README.md

```diff
@@ -159,38 +171,62 @@ function App() {
 export default App;
 ```
 
-You can pass Bridge options as a third parameter as well:
+`StoryblokComponent` here renders all the components dynamically which you loaded before during the initalization inside the `storyblokInit` function. 
+
+This is how you can pass the Bridge options as a third parameter to `useStoryblok` :
 
 ```js
-useStoryblok(story.id, (story) => (state.story = story), {
+useStoryblok(story.id, {version: 'draft'}, {
   resolveRelations: ["Article.author"],
 });
 ```
 
-#### 3. Link your components to Storyblok Visual Editor
+**Check out the React Bolierplate [here](https://github.com/storyblok/storyblok-react-boilerplate) or read on how to add Storyblok to React in 5 mins [here](https://www.storyblok.com/tp/headless-cms-react)**
 
-For every component you've defined in your Storyblok space, call the `storyblokEditable` function with the blok content:
 
-```js
-import { storyblokEditable } from "@storyblok/react";
+## Next 12 and Next 13 (no RSC)
```

Comment:
Actually, this would be pretty much the same approach to take for a Remix project. Should we add "Remix" to the title? Or create a separate section for it?

Reply:
Thinking out loud: As the App Router is stable now, and probably the "official approach", should we move this section below on the page so the devs see first the App Router approach?

Reply:
Yes! I agree to all these points! 
Also, let's add remix!

Reply:
I've made the change, please take a look.

Reply:
I haven't checked if I need to add something specific for Remix.

Reply:
Just to summarize: I discussed with @arorachakit and we will add Remix example to the README file in a separate PR, after we release the RSC support